java/ql/src/semmle/code/java/security/QueryInjection.qll

```diff
@@ -0,0 +1,48 @@
+/** Provides classes to reason about database query language injection vulnerabilities. */
+
+import java
+import semmle.code.java.dataflow.DataFlow
+import semmle.code.java.dataflow.FlowSources
```

Comment:
I think this import might be superfluous?

Reply:
`FlowSources` seem to import the `Jdbc` framework required for `SqlExpr` and `DataFlow` for `DataFlow::Node`
Do you prefer to keep `FlowSources` or the `DataFlow` import and add the `Jdbc` explictly.

I prefer the latter because it is more explicit.

Reply:
Agree. Let's keep `DataFlow`, add `Jdbc`, and remove `FlowSources`.